ext_localconf.php

```diff
@@ -36,6 +36,30 @@ function($extensionKey) {
 
         // retrieve stdWrap current value into sub cObj. CONTENT
         $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tslib/class.tslib_content.php']['postInit'][] = \JWeiland\Jwtools2\Hooks\InitializeStdWrap::class;
+
```

Comment:
Bitte nach oben in den Solr Bereich verschieben. Die se SignalSlots sollen nur dann aktiviert werden, wenn diese im ExtConf so gewünscht ist.